2200099/10505.py

```diff
@@ -0,0 +1,10 @@
+T = int(input())
+for t in range(1,T+1):
+    case = int(input())
+    pay = list(map(int, input().split()))
+    tmp_pay = sum(pay)//case
```

Comment:
```suggestion
    tmp_pay = sum(pay)/case
```
정확한 평균 값을 구하려면 `//`가 아닌 `/` 연산을 사용하시는 게 맞습니다! 😄